source/includes/api/_before_you_get_started.md

```diff
@@ -56,6 +56,134 @@ Below you can find a list of RW API resources that use the `application` field:
 * [Vocabulary](/index-rw.html#vocabulary-and-tags)
 * [Widgets](/index-rw.html#widget10)
 
+## User roles
+
+> Calling the GET /users/me endpoint providing your JWT token
+
+```shell
+curl -X GET "https://api.resourcewatch.org/auth/user/me"
+-H "Content-Type: application/json"  -d \
+-H "Authorization: Bearer <your-token>" \
+```
+
+> Response including the role field:
+
+```json
+{
+    "_id": "5dbadb0adf24534d1ad05dfb",
+    "email": "test.user@example.com",
+    "role": "USER",
+    "provider": "local",
+    "extraUserData": {
+        "apps": ["rw"]
+    },
+    "createdAt": "2019-10-31T13:00:58.191Z",
+    "updatedAt": "2019-10-31T13:00:58.191Z"
+}
+```
+
+> Example of decoded JWT token information (using jwt.io):
+
+```json
+{
+  "_id": "5dbadb0adf24534d1ad05dfb",
+  "email": "test.user@example.com",
+  "role": "USER",
+  "provider": "local",
+  "extraUserData": {
+    "apps": ["rw"]
+  },
+  "createdAt": 1597230329945,
+  "iat": 1597230329
+}
+```
+
+RW API users have a role associated with it, defined in the `role` field of each user. You can check your own role by consulting your user information using the `GET /users/me` endpoint or getting a JWT token and decoding its information. The `role` of the user is defined as a string, and it can take one of the following values:
```

Comment:
Added a link to the get current user docs and removed the examples since we already have that in the get current user docs, thanks for the suggestion :) 

I did however keep the additional information about which values the `role` can have, because we don't have that info in the user docs (we don't have a model reference there) - do you think it makes sense to add a model reference to the users? If so, we can then remove this from here and just point to the model reference!

Reply:
Having a model reference for users would make sense, but it's out of scope here, so let's not do it now.

Having the roles here: yes, I think it makes sense to have the 3 roles here, along with their description, because that will be common for all usages of roles. So, for example, we could go to dataset/widget/layer and say "there's a role param here, read the concept page for details" and the details here would hold true. Or, in other words, this section covers behavior, the endpoints cover CRUD